webapp/src/reducers/currentModal/index.ts

```diff
@@ -0,0 +1,25 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+const initialState: GlobalModalState = {
```

Comment:
this reducer was needed because when a user is disconnected and still tries to open the modal, the value of the `open` state in this reducer will still be `true`, we can't reset the value of this state while rendering the component itself, as we get an error on doing this